hub/hub-api.yaml

```
@@ -825,6 +705,30 @@ paths:
                 $ref: "#/components/schemas/ResponseError"
 
   /users:
+    post:
+      tags:
+        - Users
+      summary: Create a new user
+      description: Use this endpoint to add a new user to Inperium. Provide basic information about the new user. They will receieve an invitation by email.
```

```suggestion
      description: Use this endpoint to add a new user to Inperium. Provide basic information about the new user.
```
it is not only invite